src/Workspaces/CSharp/Portable/CSharpWorkspaceResources.resx

```diff
@@ -168,4 +168,7 @@
   <data name="Wrapping_preferences" xml:space="preserve">
     <value>Wrapping preferences</value>
   </data>
+  <data name="using_preferences" xml:space="preserve">
+    <value>using preferences</value>
```

Comment:
no comment that 'using' should be locked?

Reply:
Should `using` be in single quotes?

Reply:
@Neme12  The `var` in `var_preferences` isn't in quotes, so I think it is OK.

Reply:
one niggle there is htat 'var preferences' isn't really an english phrase.  whereas 'using preferences' is a bit ambiguous.  Is it hte preferences around 'using directives'?  Or is it 'using your preferences'?  That said, this is minor enough that i think it's fine either way.

Reply:
looking at this, i would put 'using' in quotes.